test/powershell/engine/Remoting/SSHRemotingAPI.Tests.ps1

```diff
@@ -2,6 +2,11 @@ Describe "SSH Remoting API Tests" -Tags "Feature" {
 
     Context "SSHConnectionInfo Class Tests" {
 
+        BeforeAll {
+            ## Skip the test if ssh.exe is not present.
+            $skipTest = (Get-Command 'ssh.exe' -ErrorAction SilentlyContinue) -eq $null
```

Comment:
It will skip the tests on Unix plats. Is that intentional?

Reply:
Maybe
```
$skipTest = (Get-Command 'ssh' -ErrorAction SilentlyContinue) -eq $null 
```

Reply:
Good point. Fixed.